wasm/Cargo.toml

```diff
@@ -14,6 +14,8 @@ rustpython_parser = {path = "../parser"}
 rustpython_vm = {path = "../vm"}
 cfg-if = "0.1.2"
 wasm-bindgen = "0.2"
+js-sys = "0.3"
+num-bigint = "0.2.1"
```

Comment:
This `bigint` dependency should not be required, since rustpython_vm already depends upon this package, right?

Reply:
`rustpython_vm` doesn't expose `bigint` at all (e.g. with `pub extern crate`), and trying to do `extern crate num_bigint;` didn't work before adding it to Cargo.toml.